live-server.js

```diff
@@ -15,12 +14,11 @@ var opts = {
 };
 
 var homeDir = process.env[(process.platform === 'win32') ? 'USERPROFILE' : 'HOME'];
-var configPath = path.join(homeDir, '.live-server.json');
-if (fs.existsSync(configPath)) {
-	var userConfig = fs.readFileSync(configPath, 'utf8');
-	assign(opts, JSON.parse(userConfig));
-	if (opts.ignorePattern) opts.ignorePattern = new RegExp(opts.ignorePattern);
-}
+assign(opts, readSettings(path.join(homeDir, '.live-server')));
```

Comment:
This seems to be backwards incompatible change, as the config file had .json extension before. Perhaps reading that could be added in for compatibility?

Reply:
'require' calls handle loading of json files as well. Meanig, when you call require('.live-server'); that will load js and json files. Or is there something im missing?

Reply:
Sorry, misunderstanding from my part. I wasn't aware that `require` accepts .json file extension too.

Reply:
Ok cool. I wanted to make sure i wasnt missing anything. :)